safe_cli/operators/safe_tx_service_operator.py

```diff
@@ -406,3 +411,58 @@ def drain(self, to: ChecksumAddress):
             print_formatted_text(
                 HTML("<ansigreen>Safe account is currently empty</ansigreen>")
             )
+
+    def search_account(
+        self, address: ChecksumAddress
+    ) -> Optional[Union[LocalAccount, HwWallet]]:
+        """
+        Search the provided address between loaded owners
+        :param address:
+        :return: LocalAccount or HwWallet of the provided address
+        """
+        eoa_account = [
+            account for account in self.accounts if account.address == address
+        ]
+        if eoa_account:
+            return eoa_account[0]
+
+        hw_account = [
+            account
+            for account in self.hw_wallet_manager.wallets
+            if account.address == address
+        ]
+        if hw_account:
+            return hw_account[0]
+        return None
+
+    def remove_proposed_transaction(self, safe_tx_hash: bytes):
+        eip712_message = get_remove_transaction_message(
+            self.address, safe_tx_hash, self.ethereum_client.get_chain_id()
+        )
+        message_hash = eip712_encode_hash(eip712_message)
+        try:
+            safe_tx, _ = self.safe_tx_service.get_safe_transaction(safe_tx_hash)
```

Comment:
Add a warning and requiere confirmation if the transaction have all required signatures and IS ready to sign.

Reply:
Maybe you can add this explanation as a comment

Reply:
I'm thinking to add a yes or no question is this case to ensure that the user reads the message and he is aware that burn the nonce is the best option after this.

Reply:
Added here https://github.com/safe-global/safe-cli/pull/352/commits/4cecd9c0c0a6065855aa9195dc31e726eeb53128